### ISSUE TITLE
RubyMine shows an empty window when clicked on `Show Diff` from annotations

### ISSUE THREAD

**Issue by uar-zlatko-alomerovic** [ChrisRM/material-theme-jetbrains#1426]
### Summary

RubyMine shows an empty window when clicked on `Show Diff` from annotations. The same issue happens when starting the RubyMine for the first time.

### Expected behavior

When clicked on `Show Diff` the new window should show the `git diff` between two revisions

### Actual behavior

When clicked on `Show Diff` the new window is empty and the whole UI appears empty, except for the window icons (close, minimize, maximize)

#### (**on startup**)
1. Open RubyMine

*Observe the window*:
*Expected*: The RubyMine loads UI
*Actual*: The RubyMine is only showing an empty window with window icons (close, minimize, maximize)

*To show the window contents, I use the following trick*:
1. Click on the `View` menu -> `Appearance` -> `Enter Presentation Mode`
2. Click on the `View` menu -> `Appearance` -> `Exit Presentation Mode`

#### (**when using `Show Diff`**)
1. Open any file tracked with GIT
2. Click on `VCS` menu, and from `Git` submenu select `Annotate`
3. Right-click any revision from the left pane (list of git revisions) and click `Show Diff`

*Observe the window that appears*
*Expected*: The newly opened window is empty and the window only contains icons(close, minimize, maximize)
*Actual*: The newly opened window should show the diff for the selected revision

*To get back to the open files, I just use the `ESC` button.*

The image of the issue->

<img width="1440" alt="material_theme_empty_window_2019-11-04_1258" src="https://user-images.githubusercontent.com/30901424/68120073-bbf72880-ff04-11e9-8409-5d747f4e72c0.png">

**The issue was gone when I removed the `Material Theme` plugin.**

### Environment

```
RubyMine 2019.2.4
Build #RM-192.7142.37, built on October 29, 2019
Licensed to #
You have a perpetual fallback license for this version
Subscription is active until September 11, 2020
Runtime version: 11.0.4+10-b304.77 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.13.6
GC: ParNew, ConcurrentMarkSweep
Memory: 1981M
Cores: 8
Registry: ide.mac.allowDarkWindowDecorations=true, ide.tree.ui.experimental=false, ide.balloon.shadow.size=0
Non-Bundled Plugins: com.alayouni.ansiHighlight, com.chrisrm.idea.MaterialThemeUI, com.intellij.ideolog, net.seesharpsoft.intellij.plugins.csv, org.elm.klazuka, BashSupport, com.jetbrains.lang.ejs, org.jetbrains.plugins.vue, ru.adelf.idea.dotenv
```

**material_theme.xml**
```xml
<application>
  <component name="MaterialThemeConfig">
    <option name="isContrastMode" value="true" />
    <option name="isHighContrast" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="pristineConfig" value="false" />
    <option name="selectedTheme" value="DARKER" />
    <option name="userId" value="15fd17df:166686ce982:-8000" />
    <option name="version" value="4.5.2" />
  </component>
</application>
```
Material Theme Version
<img width="250" alt="material_theme_version_2019-11-04_1257" src="https://user-images.githubusercontent.com/30901424/68119957-63279000-ff04-11e9-851c-8f5db5182aa4.png">

OS and computer Info
<img width="232" alt="macbook_info_2019-11-04_1259" src="https://user-images.githubusercontent.com/30901424/68120061-aeda3980-ff04-11e9-8d62-1aed66e2955d.png">

More references:
[IntelliJ Community Posts](https://intellij-support.jetbrains.com/hc/en-us/community/posts/360005152899-Rubymine-show-diff-empty-window?page=1#community_comment_360000908859)

**Comment by gfosteroutlier** [comment]
I also have this issue. As a temporary workaround, I've found that moving the blank `Show Diff` window (i.e. just dragging the window a few pixels any direction) restores the visibility of the contents.

**Comment by mallowigi** [comment]
This is an issue with JRE 11 and is fixed for 2019.3. In the meantime if it bothers you, you can also disable the "Themed Title bar" option and restart.

**Comment by DimaDK02** [comment]
@mallowigi, thanks for the workaround! Saved my life. BTW, the theme is beautiful, you guys are doing a great job :+1:
